applications/StructuralMechanicsApplication/README.md

```diff
@@ -1,137 +1,84 @@
+# Structural Mechanics Application
 
-## Structural Mechanics Application
-
-The Structural Mechanics Application contains a series of structural elements, as well as solid elements, constitutive laws and the corresponding strategies and solvers within Kratos Multiphysics.
+ |             **Application**             |                                                                                    **Description**                                                                                    |                              **Status**                              | **Authors** |
+|:---------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------:|:-----------:|
+| `StructuralMechanicsApplication` | The *Structural Mechanics Application* contains a series of structural elements, as well as solid elements,  the corresponding strategies, solvers and *Constitutive Laws Application* within *Kratos Multiphysics*. | <img src="https://img.shields.io/badge/Status-%F0%9F%9A%80%20Actively%20developed-Green"  width="300px"> | Riccardo Rossi *(rrossi@cimne.upc.edu)* <br /> Rubén Zorrilla Martínez *(rzorrilla@cimne.upc.edu)* <br /> Vicente Mataix Ferrándiz *(vmataix@altair.com)* <br /> Alejandro Cornejo Velázquez *(acornejo@cimne.upc.edu )*  |
```

Comment:
```suggestion
| `StructuralMechanicsApplication` | The *Structural Mechanics Application* contains a series of structural elements, as well as solid elements,  the corresponding strategies, solvers and *Constitutive Laws Application* within *Kratos Multiphysics*. | <img src="https://img.shields.io/badge/Status-%F0%9F%9A%80%20Actively%20developed-Green"  width="300px"> | Riccardo Rossi *(rrossi@cimne.upc.edu)* <br /> Rubén Zorrilla Martínez *(rzorrilla@cimne.upc.edu)* <br /> Vicente Mataix Ferrándiz *(vmataix@altair.com)* <br /> Alejandro Cornejo Velázquez *(acornejo@cimne.upc.edu )* <br /> Philipp Bucher *(philippbucher@tum.de)*  |
```

even though I think @KratosMultiphysics/structural-mechanics would be more politically correct/fairer

also please wait for the decision in #10893

Reply:
I did not include you because I didn't know which email to consider. I was expecting to consider your new position mail, but maybe was not what you want it

Reply:
> I think https://github.com/orgs/KratosMultiphysics/teams/structural-mechanics would be more politically correct/fairer

What do you mean by that?